src/styles/blocks/header.scss

```diff
@@ -0,0 +1,141 @@
+.header {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  background-image: url(../images/Header.png);
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: cover;
+
+  &__content-wrapper {
+    padding: 36px 10%;
+    display: flex;
+    flex-direction: column;
```

Comment:
add ` box-sizing: border-box` here and remove it from `*`

Reply:
done. not clear why this is the better apporach?